specs/transformation.jq

```diff
@@ -38,4 +38,6 @@
   | .paths."/cloud-connect-cspm-azure/entities/download-certificate/v1".get.parameters[1].default = "false"
 
   # Needed by rusty-falcon (stricter typing)
-  | .definitions."deviceapi.DeviceDetailsResponseSwagger".properties.errors."x-nullable" = true
\ No newline at end of file
+  | .definitions."deviceapi.DeviceDetailsResponseSwagger".properties.errors."x-nullable" = true
+
+  | .paths."/ioarules/queries/rule-groups-full/v1".get.responses."200" = .paths."/ioarules/entities/rule-groups/v1".get.responses."200"
```

Comment:
Hey @ffalor,

The PR is includes the changes to transformation, but it is missing the code change in the generated code.

Reply:
sorry about that - its added